templates/kickstart.j2

```diff
@@ -73,6 +104,17 @@ systemctl disable NetworkManager
 echo "net.ipv6.conf.all.disable_ipv6 = 1" >> /etc/sysctl.conf
 echo "net.ipv6.conf.default.disable_ipv6 = 1" >> /etc/sysctl.conf
 
-/sbin/grubby --update-kernel=`/sbin/grubby --default-kernel` --args="console=tty0 console=ttyS1,19200n8 ipv6.disable=1 numa={{ hostvars[groups[item][0]]['kernel_numa_param'] | default('on') }}"
```

Comment:
You have deprecated `kernel_numa_param`  I don't know if this was by choice or not but ceph seems to use it (I would hope that they have a good reason why they use it).

Reply:
Yes, this has also deprecated these previously hard coded kernel parameters which was the primary reason: 

> console=tty0 console=ttyS1,19200n8 ipv6.disable=1

There's an example in testing which also shows how one now can define extra grubby vars: 

https://github.com/CSCfi/ansible-role-dhcp-kickstart/blob/468307d1554291d99a83f83fa4468cd02bebb5bc/tests/group_vars/compute3/vars.yml#L14

There's also a real-world example how to continue setting numa in PR #289 in a private repo near you